hook.js

```diff
@@ -24,6 +24,9 @@ function handle(request, response) {
         logger.log(metadata);
         comment(metadata);
     }
+    else {
+        logger.logError('Invalid event received: ' + eventType);
```

Comment:
The signature for `logError` is `logError(error, title)`. Titles are printed on separate lines, so we can also safely remove the space.

I'm not sure either of those choices is intuitive, so definitely feel free to change the argument order and printing format.